README.md

```diff
@@ -10,13 +10,15 @@ enough to get you started. Here are some known limitations:
 
 * Numbered lists containing `<pre>` blocks will lose their numbering (restarting at 1 after the `<pre>`). Some complex cases will lose their list layout entirely.
 * Tables without proper headers will be rendered with an empty header
+* Multilines tables will in the first pandoc run converted mostly to HTML which we detect and then in a second might end up as proper markdown tables (if it doesn't work it stays a HTML table) by removing the `<br/>` and afterwards introducing them again into the markdown.
 * Some [interlaced formatting of inline code and bold will be messed up](https://github.com/jgm/pandoc/issues/3024)
 * `<!-- -->` may appear in final ouput in a few places because Redmine incorrectly [does not support HTML in markdown](http://www.redmine.org/issues/20497)
 
 ## Usage
 
 1. Backup your database
-2. [Install pandoc](http://pandoc.org/installing.html)
+2. [Install pandoc](http://pandoc.org/installing.html).
+   See the example script `setup.sh` and adapt it to your environment for preparing the conversion.
```

Comment:
Denoted as example script. There is also the important regex replace to allo use HTML inside markdown ...